CHANGES

```diff
@@ -98,6 +97,8 @@ Features added
 
 Bugs fixed
 ----------
+* #8522: ``__bool__`` method isn't called now by autodoc module, so it won't
+    lead to side effect or direct failure in case it generates an error.
```

Comment:
Potentially, autodoc may cause a side effect. So this changelog entry is not correct. In my understanding, this change reduces `__bool__` method calls on generating documents. It does not relate to "side effect or direct failure".

Reply:
Hello @tk0miya.

Thank you for review. I've just insured there isn't any else call like this, but you are right other side effects can occure, I will fix changelog.